--- /Users/rmills/gnu/github/exiv2/0.27-maintenance/cmake/Doxyfile.in	2019-04-25 15:26:22.000000000 +0100
+++ doxygen/Doxyfile	2019-04-25 18:55:11.000000000 +0100
@@ -51,14 +51,14 @@
 # pixels and the maximum width should not exceed 200 pixels. Doxygen will copy
 # the logo to the output directory.
 
-PROJECT_LOGO           = @ROOTDIR@/exiv2.png
+PROJECT_LOGO           = exiv2.png
 
 # The OUTPUT_DIRECTORY tag is used to specify the (relative or absolute) path
 # into which the generated documentation will be written. If a relative path is
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = doc
+OUTPUT_DIRECTORY       =
 
 # If the CREATE_SUBDIRS tag is set to YES then doxygen will create 4096 sub-
 # directories (in 2 levels) under the output directory of each output format and
@@ -780,9 +780,7 @@
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = @INCDIR@ \
-                         @SRCDIR@ \
-                         @BINDIR@/doxygen.hpp
+INPUT                  = $(EXIV2HOME)/include/exiv2 doxygen/mainpage.hpp
 
 # This tag can be used to specify the character encoding of the source files
 # that doxygen parses. Internally doxygen uses the UTF-8 encoding. Doxygen uses
@@ -807,7 +805,7 @@
 # *.m, *.markdown, *.md, *.mm, *.dox, *.py, *.pyw, *.f90, *.f95, *.f03, *.f08,
 # *.f, *.for, *.tcl, *.vhd, *.vhdl, *.ucf and *.qsf.
 
-FILE_PATTERNS          =
+FILE_PATTERNS          = *.hpp
 
 # The RECURSIVE tag can be used to specify whether or not subdirectories should
 # be searched for input files as well.
@@ -822,18 +820,18 @@
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = @ROOTDIR@/samples/Jzon.h \
-                         @ROOTDIR@/samples/Jzon.cpp \
-                         @ROOTDIR@/src/actions.hpp \
-                         @ROOTDIR@/src/actions.cpp \
-                         @ROOTDIR@/src/getopt.cpp \
-                         @ROOTDIR@/src/getopt.hpp \
-                         @ROOTDIR@/src/localtime.c \
-                         @ROOTDIR@/src/fff.h \
-                         @ROOTDIR@/src/private.h \
-                         @ROOTDIR@/src/timegm.h \
-                         @ROOTDIR@/src/tzfile.h \
-                         @ROOTDIR@/src/unused.h \
+EXCLUDE                = $(EXIV2HOME)/samples/Jzon.h \
+                         $(EXIV2HOME)/samples/Jzon.cpp \
+                         $(EXIV2HOME)/src/actions.hpp \
+                         $(EXIV2HOME)/src/actions.cpp \
+                         $(EXIV2HOME)/src/getopt.cpp \
+                         $(EXIV2HOME)/src/getopt.hpp \
+                         $(EXIV2HOME)/src/localtime.c \
+                         $(EXIV2HOME)/src/fff.h \
+                         $(EXIV2HOME)/src/private.h \
+                         $(EXIV2HOME)/src/timegm.h \
+                         $(EXIV2HOME)/src/tzfile.h \
+                         $(EXIV2HOME)/src/unused.h \
 
 
 # The EXCLUDE_SYMLINKS tag can be used to select whether or not files or
@@ -867,7 +865,7 @@
 # that contain example code fragments that are included (see the \include
 # command).
 
-EXAMPLE_PATH           = @ROOTDIR@/samples/
+EXAMPLE_PATH           = $(EXIV2HOME)/samples doxygen
 
 # If the value of the EXAMPLE_PATH tag contains directories, you can use the
 # EXAMPLE_PATTERNS tag to specify one or more wildcard pattern (like *.cpp and
@@ -1091,7 +1089,7 @@
 # The default directory is: html.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_OUTPUT            =
+HTML_OUTPUT            = $(EXIV2WEB)/html/doc
 
 # The HTML_FILE_EXTENSION tag can be used to specify the file extension for each
 # generated HTML page (for example: .htm, .php, .asp).
@@ -1118,7 +1116,7 @@
 # of the possible markers and block names see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_HEADER            =
+HTML_HEADER            = doxygen/exiv2-header.html
 
 # The HTML_FOOTER tag can be used to specify a user-defined HTML footer for each
 # generated HTML page. If the tag is left blank doxygen will generate a standard
@@ -1128,7 +1126,7 @@
 # that doxygen normally uses.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_FOOTER            =
+HTML_FOOTER            = doxygen/exiv2-footer.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading style
 # sheet that is used by each HTML page. It can be used to fine-tune the look of
@@ -1153,7 +1151,7 @@
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  =
+HTML_EXTRA_STYLESHEET  = doxygen/customdoxygen.css
 
 # The HTML_EXTRA_FILES tag can be used to specify one or more extra images or
 # other source files which should be copied to the HTML output directory. Note
@@ -1553,7 +1551,7 @@
 # The default value is: YES.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-SEARCHENGINE           = YES
+SEARCHENGINE           = NO
 
 # When the SERVER_BASED_SEARCH tag is enabled the search engine will be
 # implemented using a web server instead of a web client using Javascript. There
@@ -2104,7 +2102,7 @@
 # tag file that is based on the input files it reads. See section "Linking to
 # external documentation" for more information about the usage of tag files.
 
-GENERATE_TAGFILE       = doc/html/exiv2.xml
+GENERATE_TAGFILE       = NO
 
 # If the ALLEXTERNALS tag is set to YES, all external class will be listed in
 # the class index. If set to NO, only the inherited external classes will be
