--- __manpage1__	2018-10-26 16:21:54.000000000 +0100
+++ __manpage__	2018-10-26 16:24:44.000000000 +0100
@@ -1,13 +1,10 @@
 <HTML>
 <BODY bgcolor=pink>
-<PRE>
-<!-- Manpage converted by man2html 3.0.1 -->
-
+<H2>NAME</H2><PRE>
+       <b>exiv2</b> - <i>Image metadata manipulation tool</i>
 </PRE>
 <H2>SYNOPSIS</H2><PRE>
        <B>exiv2</B> [<I>options</I>] [<I>action</I>] <I>file</I> ...
-
-
 </PRE>
 <H2>DESCRIPTION</H2><PRE>
        <B>exiv2</B> is a program to read and write Exif, IPTC, XMP metadata and image
